src/runtime/GlobalObject.cpp

```diff
@@ -59,53 +59,16 @@ GlobalObject::GlobalObject(ExecutionState& state)
     Object::setGlobalIntrinsicObject(state);
 }
 
-void GlobalObject::installBuiltins(ExecutionState& state)
+void GlobalObject::initializeBuiltins(ExecutionState& state)
 {
     // m_objectPrototype has been initialized ahead of any other builtins
     ASSERT(!!m_objectPrototype);
 
-    installFunction(state);
-    installObject(state);
-    installIterator(state);
-    installError(state);
-    installSymbol(state);
-    installBigInt(state);
-    installString(state);
-    installNumber(state);
-    installBoolean(state);
-    installArray(state);
-    installMath(state);
-    installDate(state);
-    installRegExp(state);
-    installJSON(state);
-#if defined(ENABLE_ICU) && defined(ENABLE_INTL)
-    installIntl(state);
-#endif
-    installPromise(state);
-    installProxy(state);
-    installReflect(state);
-    installArrayBuffer(state);
-    installDataView(state);
-    installTypedArray(state);
-    installMap(state);
-    installSet(state);
-    installWeakMap(state);
-    installWeakRef(state);
-    installWeakSet(state);
-    installFinalizationRegistry(state);
-    installGenerator(state);
-    installAsyncFunction(state);
-    installAsyncIterator(state);
-    installAsyncFromSyncIterator(state);
-    installAsyncGeneratorFunction(state);
-#if defined(ENABLE_THREADING)
-    installAtomics(state);
-    installSharedArrayBuffer(state);
-#endif
-#if defined(ENABLE_WASM)
-    installWASM(state);
-#endif
-    installOthers(state);
+#define DECLARE_BUILTIN_INIT_FUNC(OBJNAME, objName, ARG) \
```

Comment:
It would be nice if you kindly add detailed comments to describe the lazy builtin initialization at the below
```c++
DECLARE_BUILTIN_INIT_FUNC 
DECLARE_BUILTIN_GETTER_FUNC
```

Reply:
Thanks for the review. I updated some comments for it.